beast/tools/run/merge_files.py

```diff
@@ -164,13 +176,20 @@ def merge_files(beast_settings_info, use_sd=True, nsubs=1):
         default=1,
         help="number of subgrids used for the physics model",
     )
+    parser.add_argument(
+        "--partial",
+        type=int,
+        default=0,
+        help="set to True (1) if fitting is partially complete and you want to merge stars that are done (relevant to subgrids only)",
+    )
 
     args = parser.parse_args()
 
     merge_files(
         beast_settings_info=args.beast_settings_file,
         use_sd=bool(args.use_sd),
         nsubs=args.nsubs,
+        partial=bool(args.partial),
```

Comment:
If a boolean is needed, why not just set this in the argparser.add_argument setup?  Wouldn't the following make sense?

`parser.add_argument("--partial", action="store_true", help="text")`

This should do the same and avoid the need for `bool` when passing to merge_files.

Reply:
That works too!  I was following the pattern from `--use_sd`, but that one is True by default, so turning it off requires using ints.